developer-docs-site/static/examples/typescript/dapp-example/src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { DappSite } from "./DappSite";
+
+function App() {
+  const aptos = (window as any).aptos;
+  const location = window.location;
+
+  const [userAddress, setUserAddress] = React.useState<string | null>(null);
```

Comment:
will be cleander just to import `import React, {useState, useEffect} from "react"`